components/emoji_picker/emoji_picker.jsx

```diff
@@ -39,88 +38,59 @@ const EMOJI_TO_LOAD_PER_UPDATE = 135;
 const SYSTEM_EMOJIS_COUNT = 1476;
 const EMOJI_LAZY_LOAD_SCROLL_THROTTLE = 100;
 
-const recentEmojiCategory = {
-    recent: {
-        name: 'recent',
-        className: 'fa fa-clock-o',
-        id: t('emoji_picker.recent'),
-        message: 'Recently Used',
+// we know some categories, but there might be new ones in an upgrade
+const categoryClass = new Map([
+    ['recent', 'fa fa-clock-o'],
+    ['searchResults', ''],
+    ['smileys-emotion', 'fa fa-smile-o'],
+    ['people-body', 'fa fa-user'],
+    ['animals-nature', 'fa fa-leaf'],
+    ['food-drink', 'fa fa-cutlery'],
+    ['activities', 'fa fa-futbol-o'],
+    ['travel-places', 'fa fa-plane'],
+    ['objects', 'fa fa-lightbulb-o'],
+    ['symbols', 'fa fa-heart-o'],
+    ['flags', 'fa fa-flag-o'],
+    ['custom', 'fa fa-at'],
+]);
+
+const DEFAULT_CLASS = 'fa fa-user';
+
+function createCategory(name) {
+    return {
+        name,
+        id: Emoji.CategoryTranslations.get(name, `emoji_picker.${name}`),
+        className: categoryClass.get(name, DEFAULT_CLASS),
+        message: Emoji.CategoryMessage.get(name, name),
         offset: 0,
-    },
-};
+    };
+}
 
-const peopleEmojiCategory = {
-    people: {
-        name: 'people',
-        className: 'fa fa-smile-o',
-        id: t('emoji_picker.people'),
-        message: 'People',
-        offset: 0,
-    },
-};
+const CATEGORIES = {};
+
+//     ...smileysEmojiCategory,
+//     'people & body': createCategory('people & body', 'fa fa-user', 'People & Body', t('emoji_picker.people-body')),
+//     'animals & nature': createCategory('animals & nature', 'fa fa-leaf', 'Animals & Nature', t('emoji_picker.animals-nature')),
+//     'food & drink': createCategory('food & drink', 'fa fa-cutlery', 'Food & Drink', t('emoji_picker.food-drink')),
+//     activities: createCategory('activities', 'fa fa-futbol-o', 'Activities', t('emoji_picker.activities')),
+//     'travel & places': createCategory('travel & places', 'fa fa-plane', 'Travel & Places', t('emoji_picker.travel-places')),
+//     objects: createCategory('objects', 'fa fa-lightbulb-o', 'Objects', t('emoji_picker.objects')),
+//     symbols: createCategory('symbols', 'fa fa-heart-o', 'Symbols', t('emoji_picker.symbols')),
+//     flags: createCategory('flags', 'fa fa-flag-o', 'flags', t('emoji_picker.flags')),
+//     custom: createCategory('custom', 'fa fa-at', 'custom', t('emoji_picker.custom')),
+// };
```

Comment:
Commented out for use later?

Reply:
no, I was comparing the new and the old one and left it behind inadvertently, I'll remove it.